.gitignore

```diff
@@ -34,3 +34,7 @@ node_modules
 
 # Optional REPL history
 .node_repl_history
+
+# VIM files
```

Comment:
I guess the VIM files is about your text editor, and there may be not need to push the and updated .gitignore, isn't it ?